index.html

```diff
@@ -4106,17 +4106,8 @@ <h4 id="adjustments_of_ruby_with_length_longer_than_that_of_the_base_characters"
           <p its-locale-filter-list="ja" lang="ja"><span class="index" id="d5e12127"><a class="characterClass" href="#cl-05">中点類（cl-05）</a></span>にも最大でルビ文字サイズの全角までルビ文字を掛けてもよい．ただし，中点類の前後の四分アキが行の調整処理で詰められている場合は，前にくる中点類の場合は中点類の後ろの空き量＋ルビ文字サイズの二分まで，後ろにくる中点類の場合は中点類の前の空き量＋ルビ文字サイズの二分までである．</p>
         </li>
         <li id="id237">
-          <p its-locale-filter-list="en" lang="en"> When the adjacent character is one of the <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, the ruby text may go over the base characters up to the full-width size of a ruby character. Note that the overhang must not go beyond the closing bracket itself. </p>
-          <p its-locale-filter-list="ja" lang="ja">後ろにくる<span class="index" id="d5e12151"><a class="characterClass" href="#cl-02">終わり括弧類（cl-02）</a></span>には，最大でルビ文字サイズの全角までルビ文字を掛けてもよい．この場合，終わり括弧類の後ろの空き量に掛けてはならない．</p>
-        </li>
-        <li id="id238">
-          <p its-locale-filter-list="en" lang="en"> When the adjacent character is a <a class="characterClass" href="#cl-07">comma (cl-07)</a> or <a class="characterClass" href="#cl-06">full stop (cl-06)</a>, the ruby text may go over the base characters and overhang the comma or full stop up to the full-width
-            size of a ruby character. Note that the overhang must not go beyond the comma or the period itself. </p>
-          <p its-locale-filter-list="ja" lang="ja">後ろにくる<span class="index" id="d5e12174"><a class="characterClass" href="#cl-06">句点類（cl-06）</a></span>又は<span class="index" id="d5e12178"><a class="characterClass" href="#cl-07">読点類（cl-07）</a></span>には，最大でルビ文字サイズの全角までルビ文字を掛けてもよい．この場合，句点類又は読点類の後ろの空き量に掛けてはならない．</p>
-        </li>
-        <li id="id239">
-          <p its-locale-filter-list="en" lang="en"> Also, when the adjacent character is one of the <a class="characterClass" href="#cl-01">opening brackets (cl-01)</a> before the ruby object, the ruby text may go over the base characters and hang over the opening brackets up to the full-width size of a ruby character. Note that the overhang must not go beyond the opening brackets.</p>
-          <p its-locale-filter-list="ja" lang="ja">前にくる<span class="index" id="d5e12202"><a class="characterClass" href="#cl-01">始め括弧類（cl-01）</a></span>にも，最大でルビ文字サイズの全角までルビ文字を掛けてもよい．この場合，始め括弧類の前の空き量に掛けてはならない．</p>
+          <p its-locale-filter-list="en" lang="en">When the adjacent character is one of the <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, <a class="characterClass" href="#cl-06">full stop (cl-06)</a> or <a class="characterClass" href="#cl-07">comma (cl-07)</a> after the ruby object, or one of the <a class="characterClass" href="#cl-01">opening brackets (cl-01)</a> before the ruby object, the ruby text may go over the base characters up to the full-width size of a ruby character. Note that the overhang must not go beyond the closing bracket, the full stop, the comma or the opening bracket itself. </p>
```

Comment:
<p its-locale-filter-list="en" lang="en">When the adjacent character is one of the <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, <a class="characterClass" href="#cl-06">full stop (cl-06)</a> or <a class="characterClass" href="#cl-07">comma (cl-07)</a> after the ruby object, or one of the <a class="characterClass" href="#cl-01">opening brackets (cl-01)</a> before the ruby object, the ruby text may overhang the adjacent base character up to the full-width size of a ruby character. Note that the overhang must not go beyond the closing bracket, the full stop, the comma or the opening bracket itself. </p>

Reply:
For ease of review, @macnmm's comment changes

"the ruby text may **go over** the base **characters**" to

"the ruby text may **overhang** the **adjacent** base **character**".